src/config.cpp

```diff
@@ -59,6 +66,106 @@ void parse_uint32(void* value_ptr, const char* txt) {
     cfgval->set = true;
 }
 
+// Helper function to trim whitespace and remove quotations
+std::string trim(const std::string& str) {
+    size_t first = str.find_first_not_of(" \t\n\""); // Also include quotes
+    size_t last = str.find_last_not_of(" \t\n\"");
+
+    // Check for valid range
+    if (first == std::string::npos || last == std::string::npos) {
+        return str; // Empty or only whitespace/quotes
+    }
+
+    // Extract the trimmed and unquoted substring
+    std::string trimmed = str.substr(first, (last - first + 1));
+
+    // Check if the trimmed string still starts and ends with quotes
+    if (trimmed.front() == '"' && trimmed.back() == '"') {
+        return trimmed.substr(1, trimmed.size() - 2); // Remove the quotes
+    } else {
+        return trimmed;
+    }
+}
```

Comment:
I'm not sure I understand why we need to handle quotes at all. Can you explain?

Reply:
good point. I thought some people may try to put quotes around the confs. Removing this functionality.

Reply:
It feels like this functionality has not been removed, am I missing something?